tools/prepare-built-branch.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+if [ ! -f "class.jetpack.php" ]; then
+	echo "This must be executed from the /projects/plugins/jetpack/ dir."
```

Comment:
It should probably be moved into the Jetpack plugin then.

OTOH, maybe it can be genericized if we remove all the "add custom stuff" and .svnignore in favor of defining them as export-ignore in .gitattributes. Meaning all that would be left would be the equivalent of
```
sed -i -n -e '1,/^## Things we will need in release branches$/p' .gitignore
```

Reply:
Yeah, I can move it for now. I'd love to be able to abstract out as much of this as possible so we can have some uniform build commands repo-wide, but... that can be a later project. I'll leave the check there because it's true either way—-the script needs to be run from the plugin root and that wasn't checked before.

Reply:
6b571f728